ComponentScreens/Tab/Home/TrendingMovies.js

```diff
@@ -0,0 +1,132 @@
+import React, {Component} from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+  Image,
+  FlatList,
+  TouchableOpacity,
+  ImageBackground,
+  ScrollView,
+} from 'react-native';
+import bgCover from '../../../assets/bgimage.png';
+
+export default class Trending extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      moviesTrending: [],
+    };
+  }
+
+  componentDidMount() {
+    this.fetchTrendingMovies();
+  }
+
+  fetchTrendingMovies() {
```

Comment:
This method was seen before as well. Make it reusable. Don't copy and create again.